en/deploy-tidb-operator.md

```diff
@@ -123,7 +123,7 @@ After creating CRDs in the step above, there are two methods to deploy TiDB Oper
     {{< copyable "shell-regular" >}}
 
     ```shell
-    helm upgrade tidb-operator pingcap/tidb-operator -f  ${HOME}/tidb-operator/values-tidb-operator.yaml
+    helm install tidb-operator ./tidb-operator --namespace=tidb-admin --create-namespace
```

Comment:
```suggestion
    helm upgrade tidb-operator pingcap/tidb-operator -f  `${HOME}/tidb-operator/values-tidb-operator.yaml`
```

Reply:
The Chinese version does not have wrapping backticks.
![image](https://user-images.githubusercontent.com/41534398/104873049-87954e00-598a-11eb-9def-1f19ce0a52d1.png)